uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -488,11 +481,16 @@ define([
     function setCalendarProps(contactID, calendar) {
       var dateObj,
         leaveRequest,
-        dates = Object.keys(calendar.days);
+        dates = Object.keys(calendar.days),
+        indexedByMonth = angular.copy(vm.months.map(function () {
```

Comment:
You are returning an array of arrays? `[[], [], [], []]`? What's the need for `angular.copy`? Can't it be just
```js
indexedByMonth = vm.months.map(function () { return []; });